EIPS/eip-unified_documentation.md

```diff
@@ -0,0 +1,41 @@
+---
+eip: <to be assigned>
+title: Unified Documentation
+author: Micah Dameron(@chronaeon)
+Discussions-to: https://github.com/ethereum/EIPs/issues/1054 
+status: Draft
+type: Administrative
+category: Emergent
+created: 2018-05-03
+---
+
+## Simple Summary
+Combine official Ethereum documentation into a unified reference that is maintained frictionlessly by the community.
+## Abstract
+We propose a comprehensive tagging system that signals sourcing on statements about Ethereum in a Wiki-based environment.
+## Motivation
+There are a variety of types and kinds of documentation about Ethereum; from the complete Solidity documentation, to the Ethereum Wiki, Whitepaper, Yellowpaper, to core-developer blogs, to comments inside Geth and other codebases, to articles on Medium, conversations in Gitter, and still more sources that remain unmentioned. 
+Despite these thorough and quality sources of individual documentation, the overall state of Ethereum’s documentation is in many ways undeveloped. Like slivers of cracked glass, there are multiple reflections where one expects to see a single, comprehensive image. This causes difficulties on three layers:
+Outsiders have trouble getting a unified view of Ethereum.
+Client teams each work from slightly altered points of view.
+Insiders can have different perspectives from each other about what is most important, depending on their primary sources for information about Ethereum.
+## Specification
```

Comment:
I don't think this specifies anything that can be standardised with an EIP. If you want to build this, go ahead!